src/signer_node/message_processor/process_completedblock.rs

```diff
@@ -17,6 +17,16 @@ where
         return prev_state.clone(); // Ignore message
     }
 
+    if let Err(e) = params.rpc.submitblock(block) {
+        log::warn!(
```

Comment:
Failures in submitting the block is not a rare case.
I think the state can transit to RoundComplete even if 'submitblock' fails.

Reply:
You are right, thanks.
I will update.

Reply:
fixed at 4aaa06237da4b9f9bbbcd5e53e7889a99783b4ba